kingpin/actors/aws/cloudformation.py

```diff
@@ -1080,6 +1112,23 @@ def _diff_params_safely(self, remote, local):
 
         return False
 
+    def _template_body_with_hash(self) -> str:
+        """Add a hash to the template to force a change in the stack."""
+
+        template_obj = json.loads(self._template_body)
+
+        if not isinstance(template_obj.get("Outputs", None), dict):
+            # overwrite the outputs with an empty dict
+            template_obj["Outputs"] = {}
+
+        template_obj["Outputs"][KINGPIN_CFN_HASH_OUTPUT_KEY] = {
+            "Value": md5(
+                datetime.datetime.now(datetime.timezone.utc).isoformat().encode()
```

Comment:
What are we doing here? This value will change on every deploy. I thought you were going to hash the template contents?